docs/GETTING-STARTED.md

```diff
@@ -34,6 +37,10 @@ app.get('/api-docs.json', function(req, res) {
 });
 ```
 
+- `options.jsDocFilter` is a function which accepts only one variable `jsDocComment`. This `jsDocComment` represents each route documentation being iterated upon.
+  
+  If you want to optionally perform filters on each route documentation, return boolean `true` or `false` accordingly on certain logical conditions. This is useful for conditionally displaying certain route documentation based on different server deployments.
```

Comment:
As mentioned in the comment above, I'd personally prefer to keep boolean and object types separate in terms of logic. For instance, `jsDocFilter` can be simply a reducer function. If the function is set, it's obviously `true` :)